docs/en/latest/plugins/limit-conn.md

```diff
@@ -93,17 +99,19 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
 }'
 ```
 
-You also can complete the above operation through the web interface, first add a route, then add limit-conn plugin:
-![enable limit-conn plugin](../../../assets/images/plugin/limit-conn-1.png)
+<!-- ![enable limit-conn plugin](https://raw.githubusercontent.com/apache/apisix/master/docs/assets/images/plugin/limit-conn-1.png) -->
 
-## Test Plugin
+## Example usage
 
-The parameters of the plugin enabled above indicate that only one concurrent request is allowed. When more than one concurrent request is received, will return `503` directly.
+The example above configures the Plugin to only allow one concurrent request. When more that one request is received, the Plugin will respond with a 503 status code:
 
 ```shell
 curl -i http://127.0.0.1:9080/index.html?sleep=20 &
 
 curl -i http://127.0.0.1:9080/index.html?sleep=20
+```
+
+```shell
```

Comment:
I'm not sure if using `shell` here is properly :)

Reply:
I agree with Zhiyuan, `shell` is not recognized by the renderer used by the Docusauraus, although it is supported in GitHub. Refer https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js#L8 and https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L5837. Use `bash` instead.